default_gateway.go

```diff
@@ -35,15 +37,9 @@ func (sb *sandbox) setupDefaultGW(srcEp *endpoint) error {
 		return nil
 	}
 
-	n, err := c.NetworkByName(libnGWNetwork)
```

Comment:
Can we please avoid getting into serialization and creating a network if it is already present?